src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="all--stars">
+      <div class="stars stars--0">
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="gold__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+      </div>
+
+      <div class="stars stars--2">
+        <div class="gold__star"></div>
+        <div class="gold__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+        <div class="gray__star"></div>
+      </div>
+
+      <div class="stars stars--3">
+        <div class="gold__star"></div>
```

Comment:
Check the task requirement about class naming:
<img width="1440" alt="Screenshot 2023-02-23 at 17 45 59" src="https://user-images.githubusercontent.com/52370890/220958006-0e0d1692-32ed-40e2-ab0a-a658f8857989.png">